themes/digital.gov/src/scss/new/shortcodes/_cspan-iframe.scss

```diff
@@ -0,0 +1,14 @@
+.cspan-embed {
+  iframe {
+    aspect-ratio: 4/3;
+    height: 100%;
+    width: 100%;
+  }
```

Comment:
You could remove these styles by using USWDS's aspect class `add-aspect-4x3`.

Reply:
Replaced with utility class

Reply:
Thank you! Resolved in [6944183](https://github.com/GSA/digitalgov.gov/pull/5827/commits/6944183eeac8a0d3ca786fe71927a51e4f0866fe).